src/ai/backend/manager/models/gql.py

```diff
@@ -289,6 +289,7 @@ class Queries(graphene.ObjectType):
         Agent,
         scaling_group=graphene.String(),
         status=graphene.String(),
+        deprecation_reason="deprecated_reason",
```

Comment:
When talking about the deprecation it is important to also mention about alternative solutions. Please update every deprecation reason comments to do so.